src/python/pants/option/global_options.py

```diff
@@ -329,6 +333,12 @@ def register_bootstrap_options(cls, register):
              help='Whether or not to cleanup directories used for local process execution '
                   '(primarily useful for e.g. debugging).')
 
+    # TODO: rename this to --include-dependees when the corresponding --changed-include-dependees is
+    # fully deprecated!
+    register('--include-dependees-behavior', type=IncludeDependees, default=IncludeDependees.none,
```

Comment:
So, I think that making this option global has some interesting implications that are not really accounted for here.

A user would likely assume that this option would affect _all_ commands, rather than only `changed` and `dependees`. And doing that would require more refactoring in `TargetRootsCalculator`. And it may or may not be the best time for that refactoring given the `--query` proposal.